README.md

```diff
@@ -36,6 +36,33 @@ To see the graphics, you can open another terminal and run
 GAZEBO_MODEL_PATH=`pwd`/../python/social_bot/models gzclient
 ```
 
+## Environments and Tasks
+
+### [Simple Navigation](python/social_bot/envs/simple_navigation.py)
+
+* [Goal task](python/social_bot/teacher_tasks.py). 
+    
+    To be updated
+
+### [Grocery Ground](python/social_bot/envs/grocery_ground.py)
+
+* [Goal task](python/social_bot/teacher_tasks.py). 
```

Comment:
Linking to grocery_ground.py might be better. teacher_tasks.py here does not clearly tell what the task is.

Reply:
yes.